spec/models/cma_case_spec.rb

```diff
@@ -73,7 +51,7 @@ def cma_case_content_item(n)
       "description" => "This is the summary of example CMA case 0",
       "link" => "/cma-cases/example-cma-case-0",
       "indexable_content" => "## Header" + ("\r\n\r\nThis is the long body of an example CMA case" * 10),
-      "public_timestamp" => "2015-11-16T11:53:30+00:00",
+      "public_timestamp" => "2015-12-03T16:59:13+00:00",
```

Comment:
sadly it does, because some of the expectations tests the whole payload and it needs to match the stub on line 4